scripts/install-containerd

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# install and start containerd daemon
+
+set -eu -o pipefail
+
+ARCH="$(go env GOARCH)"
+OS="$(go env GOOS)"
+CONTAINERD_VERSION="$(go list -f {{.Module.Version}} -find github.com/containerd/containerd)"
```

Comment:
This will yield `v1.5.18`, so we need to cut the first character for the curl to work correctly.